oauth_dropins/appengine_config.py

```diff
@@ -90,6 +91,7 @@ def read(filename):
 FLICKR_APP_SECRET = read('flickr_app_secret')
 GOOGLE_CLIENT_ID = read('google_client_id')
 GOOGLE_CLIENT_SECRET = read('google_client_secret')
+INDIEAUTH_CLIENT_ID = read('indieauth_client_id')
```

Comment:
No definitely not secret, but I was thinking if users didn't want to use the file method, they could just call

```python
appengine_config.INDIEAUTH_CLIENT_ID = '...'
```

in their application. (Otherwise I'm not really sure where it would make sense to put it, since you as a user never directly construct StartHandler and CallbackHandler)